ynput/OpenPype
# Blender: Fix pointcache family and fix alembic extractor

## Changelog Description
Fixed `pointcache` family and fixed behaviour of the alembic extractor.

## Additional info
The `pointcache` family was implemented in an old way. This PR changes the creator for `pointcache` family to be compatible with the alembic extractor.

The alembic extractor has also been changed to be mandatory for the `pointcache` family, but optional for models. The extraction of models as alembic is now active by default.

## Testing notes:
For the fix to `pointcache` family:
1. Try creating a new `pointcache` instance.
2. Try to publish it.

For the changed alembic extractor for `models`:
1. Try publish a (new or already existing) model.
2. There is a new extractor "Extract Model ABC". It should be active by default, optional and if active, it should publish the model as alembic.
